_posts/2018-11-16-JOIN-2018.md

```diff
@@ -0,0 +1,534 @@
+This will help them focus and deliver a better solution.
+
+Focus on what truly matters for your product.
+
+### Questions
+#### The product owner has a lot of hats, how can he manage all this?
+The product owner is indeed responsible for a lot of stuff.
+
+But being responsible does not mean that you need to deliver everything yourself. 
+Learn to delegate.
+
+The product owner should also not be the only one who meets the stakeholders and not everybody should have to be a stakeholder.
+Learn to remove stakeholders who do not contribute to the business value.
+Remember the Frankenstein monster.
+
+#### Can a product owner ignore stakeholders wishes?
+A Product Owner should be able to take his own decisions and decide to run some tests on customers and collect the resulting data.
+
+It is sometimes better to ask for forgiveness later.
+
+If the product owner has no trust of the organization making him/her unable to take decisions, then that person should not be the product owner.
+
+> You can rewatch her [talk](https://www.youtube.com/watch?v=edI6wDzIo8k){:target="_blank"} on our [channel](https://www.youtube.com/channel/UCsebfWdqV7LqNNDMDvCESIA){:target="_blank"}.
+
+## Deep learning in practice by Remco Runge
+<span class="image left"><img class="p-image" alt="Remco Runge" src="/img/join-2018/remco-speaker-zoom.jpg"></span>
+
+Machine learning: you try to get the good algorithm to find the stuff you want to find.
+
+Deep Learning is based on biology, about how people thought the brain functioned. 
+The assumption was proven wrong but the basic principles still work.
+
+Tensorflow has a nice [playground](https://playground.tensorflow.org/){:target="_blank"} where you can see these effects live in action.
+
+Deep learning has two flows:
+* Forward propagation: feed data through it.
+* Backward propagation: update weights so the next iteration will be better.
+
+You can define weights on input in order to make certain inputs more important.
+
+Deep learning requires a very large network with a large amount of nodes and a lot of hidden layers.
+
+The early layers allow your network to distinguish basic structures within your data.
+Further down in the layers the network will learn more detailed features like; eyes, ears, ... 
+
+It is cool that the network learns these features about how to recoginize a person just by providing it with pictures of faces. 
+We did not provide it with the concept `eyes` but eventually, in one of the iterations the network started to take eyes into consideration.
+
+### Deep Learning at Ordina
+
+#### Workshop: hack a drone.
+A small drone was equiped with deep learning to detect and identify objects by using the [Deep Learning for Java](https://deeplearning4j.org/){:target="_blank"}.
+In a short demo we saw it could identify a notebook.
+
+#### Tic Tac Toe
+In contrary to [Go](https://en.wikipedia.org/wiki/AlphaGo){:target="_blank"}, Tic Tac Toe just has 360.000 possible moves making it easier to use for a demo.
```

Comment:
This link still points to the AI, shouldn't it point to the Go bord game instead?
```suggestion
In contrary to [Go](https://en.wikipedia.org/wiki/Go_(game)){:target="_blank"}, Tic Tac Toe just has 360.000 possible moves making it easier to use for a demo.
```